Pod/Classes/LiquidFloatingActionButton.swift

```diff
@@ -240,6 +241,7 @@ public class LiquidFloatingActionButton : UIView {
     }
 
     private func didTapped() {
+        delegate!.didTapped!(self)
```

Comment:
Both the delegate and the protocol method are optionals, the force unwrapping on this line will lead to crashes if either the delegate is not provided or the protocol method didTapped: is not implemented. 

Use ? instead of !